themes/bootstrap3/templates/RecordTab/reviews.phtml

```diff
@@ -13,7 +13,13 @@
           <?php if (isset($review['Rating'])): ?>
             <img src="<?=$this->imageLink($review['Rating'] . '.gif')?>" alt="<?=$review['Rating']?>/5 Stars"/>
           <?php endif; ?>
-          <strong><?=$review['Summary']?></strong> <?=isset($review['Date']) ? strftime('%B %e, %Y', strtotime($review['Date'])) : ''?>
+          <strong><?=$review['Summary']?></strong>
+          <?php
+            if (isset($review['Date'])) {
```

Comment:
Why are we not using the normal display date format (from config.ini) here?

Reply:
I think the answer to both of your questions is "this code was written before we had a normal display date format," because anything using strftime is very old, coming from pre-1.0 VuFind originally.

In this particular case, I think perhaps it is justified to diverge from the standard date format, since to my mind, the date on a book review designed for long-form reading shouldn't necessarily resemble the date format for due dates -- using a spelled-out, internationalized version makes for more comfortable reading.

That being said, I'm not so strongly attached to the idea that I'm unwilling to standardize this. I'm also open to adding a method to the DateTime view helper to encapsulate the `IntlDateFormatter` logic if you think it's worth keeping but want to make it cleaner/more readable/more reusable.

Reply:
I think we need `longDisplayDateFormat` setting or such for this. It's quite possible that the desired format is not the same as the system locale's format. Personally I find long formats harder to understand, but you're right in that in this case it could work.

Reply:
It's kind of tricky because you have a choice of configuring a date format which is very specific, but which may not work equally well for all locales, or using a locale-friendly date, which only offers you vague options like long/medium/short. Perhaps we should extend the VuFindDate class so that for special values of `$outputFormat`, it uses the `IntlDateFormatter` class instead of `DateTime::format()`. Then, as you say, we could have a separate config.ini setting to control long-form display dates, and everything could be made more uniform and configurable.

That being said, it's an awful lot of work (and requires some significant design decisions) in order to change something that currently affects only one rarely-seen template (now that we've dropped Amazon support, the only thing using this is Guardian reviews, and I doubt many people are turning those on). How would you feel about leaving this as-is for now but opening a JIRA ticket to capture the idea for future reference?